src/templates/_page_list.php

```diff
@@ -4,10 +4,12 @@
 
 ?>
 
-<div id="spw-page-list">
-  <ul>
+<div id="spw-page-list" class="mole-sidebar-object-div">
+  <ul class="mole-sidebar-object-ul">
     <?php foreach ($pages as $page): ?>
-      <li><?php echo Helpers::build_sidebar_link($page) ?></li>
+    <li class="mole-sidebar-object <?php 
+      if((wp_get_post_parent_id( $post_ID ) == $page->ID) || ((get_the_ID() == $page->ID) && (wp_get_post_parent_id( $post_ID ) == 0 ) ) ) { echo ' mole-sidebar-parent ';}else if(get_the_ID() == $page->ID){ echo ' mole-sidebar-active ' ;} ?>" >
```

Comment:
How can we abstract this logic into a helper method?  I'd like to avoid embedding any logic (especially conditional logic) in the HTML directly where ever possible.